README.md

```diff
@@ -15,21 +15,24 @@ and can be run against a local server, e.g.:
 
  * Rust stable
  * MySQL 5.7 (or compatible)
-   * libmysqlclient (brew install mysql on macOS, apt-get install libmysqlclient-dev on Ubuntu)
+   * libmysqlclient (`brew install mysql` on macOS, `apt-get install libmysqlclient-dev` on Ubuntu)
 
 ## Setting Up
 
-1) [Install Rust]
-
-2) Create a `syncstorage` user/database
+1) [Install Rust](https://www.rust-lang.org/tools/install)
```

Comment:
Could also use `https://rustup.rs.`

Reply:
ah, sure. I can change it if that's the preferred way to install?

Reply:
Sadly, depends on who you ask. Go with your original URL for now. If we hear enough folk say otherwise, we can change it.

Reply:
Personally I'd say use rustup.